nucypher/config/base.py

```diff
@@ -834,7 +832,7 @@ def configure_pre_payment_method(self):
         if pre_payment_class.ONCHAIN:
             # on-chain payment strategies require a blockchain connection
             pre_payment_strategy = pre_payment_class(
-                domain=self.taco_domain_info.name,
+                domain=str(self.domain),
```

Comment:
Any reason we don't want to just pass the `TACoDomain` object instead of the str?

Reply:
I must have missed this one in the refactor -- I'll take a look!

Reply:
✔️